consensus/ConsensusFileGenerator.py

```diff
@@ -165,6 +172,18 @@ def _get_matching_history(self, variant):
                 variant_history = self._add_history_of_variant(possible_id + '_dup0', export, variant_history)
                 variant_history = self._add_history_of_variant(possible_id + '_dup1', export, variant_history)
 
+            alternative_history = self.alternative_history[export_id]
+            if 'transcript' in variant and 'c_dna' in variant:
+                variant_id = self._check_alternative_history(variant['transcript'], variant['c_dna'], gene,
+                                                             alternative_history)
+                if variant_id and variant_id not in variant_history:
+                    variant_history.append(variant_id)
+                    message = '{} is invalid; will be replaced by correct variant {} in future releases\n'.format(
```

Comment:
if I run the molgenis-py-consensus pipeline then the incorrect variant will be replaced with a new correct variant (or merged with an existing correct variant) directly, right? so why do you refer to future releases here?

Reply:
Yes, the message does look a bit off. I'll change this message.